cpp/util/openssl_scoped_types.h

```diff
@@ -44,6 +44,7 @@
 #include <openssl/x509.h>
 #include <openssl/x509v3.h>
 
+namespace util {
```

Comment:
Would still leave `ssl_client.h`, `test_signer.cc`, `sparse_merkle_tree_test.cc` needing some `using` or qualifiers, in addition to the three header files, so 6 files affected.